rskj-core/src/test/resources/dsl/453_rollback_during_creation/exceeding_code_size

```diff
@@ -0,0 +1,66 @@
+
+comment
+
+// Contracts compiled using 0.8.20
+// the contract source code
+
+contract BytecodeFactory {
+    event ContractDeployed(address contractAddress);
```

Comment:
question:

I see that you are emitting this event but not verifying it in the DSL test. You could check the emitted event like [this](https://github.com/rsksmart/rskj/blob/314708cf08c792080ba02c90015e3ecd477cccad/rskj-core/src/test/java/co/rsk/vm/opcode/TransientStorageDslTest.java#L288). Or, just delete the emit event contract part. It doesn't seem to bring value.

Reply:
Removed the emit event for the contract, as it’s not being used or checked. :)